src/python/grpcio/grpc/_cython/_cygrpc/call.pyx.pxi

```diff
@@ -30,9 +30,12 @@ cdef class Call:
         tag, operations, self if retain_self else None)
     batch_operation_tag.prepare()
     cpython.Py_INCREF(batch_operation_tag)
-    return grpc_call_start_batch(
+    cdef grpc_call_error error
+    with nogil:
+      error = grpc_call_start_batch(
```

Comment:
nit:  We should be able to just ```return``` out of the ```with``` statement, so we don't need to store ```error``` in a temporary variable (unless cython doesn't support that syntax, in which case disregard).

Reply:
Cython doesn't allow that. If return in the with statement, we will get:
Converting to Python object not allowed without gil.